### PR TITLE
Create example app

### PR DESCRIPTION
**Pull request by rudigiesler**
Creates an example app that fetches an  XForm from Ona, processes it, and submits the results back to Ona.

### CODE REVIEW

**Comment by rudigiesler** [comment]
@hodgestar Ready for review.

File: examples/test/fixtures.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                    answer: "Jon Snow",
+                },
+            },
+            response: {
+                code: "200",
+                headers: {
+                    "Content-Type": "application/json",
+                },
+                // TODO: Find out why it is necessary to stringify
+                // or if I'm doing something wrong
```
Comment (hodgestar) [inline examples/test/fixtures.js]: I think headers need to be a list, e.g. `"Content-Type": ["application/json"]`.

Comment (hodgestar) [inline examples/test/fixtures.js]: From the docs:

```
:param object opts.request.headers:
        An object mapping each header name to an array of header values.
```

File: examples/test/fixtures.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+                    "Content-Type": "application/json",
+                },
+                data: {
+                    answer: "20",
+                },
+            },
+            response: {
+                code: "200",
+                headers: {
+                    "Content-Type": "application/json",
```
Comment (hodgestar) [inline examples/test/fixtures.js]: Hmm. I don't think our fixture stuff supports response headers (an oversight on our part probably).

**Comment by hodgestar** [comment]
Left some small comments about tests, otherwise looks great.

**Comment by rudigiesler** [comment]
@hodgestar Comments addressed

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**